apps/designer/app/routes/designer/$id.tsx

```diff
@@ -33,19 +37,46 @@ type Error = {
   errors: "string";
 };
 
+const deleteAssetSchema = zfd.formData({
+  assetId: zfd.text(),
+  assetName: zfd.text(),
+});
+
 export const action: ActionFunction = async ({ request, params }) => {
   if (params.id === undefined) throw new Error("Project id undefined");
+  try {
+    if (request.method === "DELETE") {
+      const { assetId, assetName } = deleteAssetSchema.parse(
+        await request.formData()
+      );
+      const deletedAsset = await deleteAsset({
+        id: assetId,
+        name: assetName,
+        dirname: __dirname,
```

Comment:
tbh I don't even understand why we need __dirname, which is the routes/designer in this case ... I would need to follow this code through but on the surface it doesn't make sense, because that folder where we handle the route should not be used at all